frontend/src/views/Evaluation/Nomination/NominationTable.tsx

```diff
@@ -39,24 +57,34 @@ const columns: DataTableColumn<DataTableItem>[] = [
         label: 'Organization',
         sortable: false,
     },
+    {
+        key: 'delete',
+        accessor: 'participant',
+        label: '',
+        component: DeleteColumn,
+        priority: 3,
+    },
 ]
 
 interface NominationTableProps {
     participants: Participant[]
+    progressionStatus: ProgressionStatus
```

Comment:
Is it better to call it (?)
```suggestion
    currentProgressionStatus: ProgressionStatus
```